Controller/GeneratorController.php

```diff
@@ -114,9 +113,12 @@ private function getMode(Request $request, $mode = null)
      */
     private function buildForm(PasswordGeneratorInterface $passwordGenerator, Options $options, $mode = '')
     {
-        return $this->createForm(new OptionType($passwordGenerator), $options, array(
-            'action' => $this->generateUrl('hackzilla_password_generator_show', array('mode' => $mode)),
+        return $this->createForm('Hackzilla\Bundle\PasswordGeneratorBundle\Form\Type\OptionType', $options, array(
```

Comment:
If you bump minimum PHP requirement to 5.5, you could use `OptionType::class` that is nicer to read.